lib/index.ts

```diff
@@ -237,7 +227,7 @@ export type MaplibreGeocoderApiConfig = {
 
 export type MaplibreGeocoderFeatureResults = { type: "FeatureCollection", features: CarmenGeojsonFeature[]};
 export type MaplibreGeocoderSuggestionResults = { suggestions: { text: string, placeId?: string }[] };
-export type MaplibreGeocoderPlaceResults = { place: CarmenGeojsonFeature[] };
+export type MaplibreGeocoderPlaceResults = { place: CarmenGeojsonFeature };
```

Comment:
Shouldn't this improve the types in the code to avoid "as any" somewhere? Why doesn't this solve or break anything?

Reply:
Yes, I see that this broke the ```MaplibreGeocoderResults``` used by _handleGeocodeResponse which is expecting all of the geocoder function results to return arrays either directly or as the value of a property of the returned object. 
I've reverted that and fixed the original issue by removing the array that was wrapped around ```getPlacebyId```'s `place` array on line . The extra array caused _handleGeocodeResponse to throw an error and the map to never zoom on the selected place.